ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -44,9 +45,9 @@ export class LoggingService {
         return Object.keys(obj).map(key => `${key}: ${obj[key]}`).join(', ');
     }
 
-    public logToCloud(payload: string, labels?: {[key: string]: string}, severity = 'INFO'): void {
+    public logToCloud(payload: string, labels?: {[key: string]: string}, severity = 'INFO'): Observable<void> {
         if (!this.config.doCloudLogging) {
-            return;
+            return of();
```

Comment:
Seems, this (`of()`) will never emit. 
I believe, using `of(null)` would be proper there.

Reply:
P.S. After debugging I see that `of()` completes the Observable even though [Rxjs documentation](https://rxjs.dev/api/index/function/of) teels us it shoud behave like `Never`

![image](https://user-images.githubusercontent.com/7396837/142859215-6ccfbc62-cf37-4020-898f-cf889501e3e9.png)

Reply:
Interesting!  Found same problem. I think I fixed this now.
bce5f34e028a5e449101e7d37f61baf52c85e266